source/release-notes/1.8.txt

```diff
@@ -21,16 +21,21 @@ Patch Releases
 .. _1.8.1-c2c-release-notes:
 
 1.8.1 Release
--------------
+~~~~~~~~~~~~~
 
 **October 10, 2024**
 
-mongosync Authentication with Atlas Workload Identity Federation 
-----------------------------------------------------------------
+New Features:
 
-.. include:: /includes/mongosync-and-oidc.rst
+- Support added for OpenID Connect (OIDC) authentication with
+  the workload (machine) flow. For more information, see
+  :ref:`c2c-rn-atlas-workload-identity`.
 
-For details, see :ref:`c2c-authentication`.
+Issues Fixed:
+
+- Fixed bug introduced in ``mongosync`` 1.8.0, where
+  ``mongosync`` crashes if it finds a replica set with 
+  :rsconf:`settings.replicaSetId` unset.
```

Comment:
Is it worth linking to the SERVER ticket that caused this? Just so users can have context on whether the issue affects them.

Reply:
@FGasper, Generally, we don't link to the JIRA tickets.  But, if you think it's important, I can throw it in.